app/src/main/java/org/listenbrainz/android/ui/navigation/AppNavigation.kt

```diff
@@ -44,6 +44,7 @@ fun AppNavigation(
     scrollRequestState: Boolean,
     onScrollToTop: (suspend () -> Unit) -> Unit,
     snackbarState: SnackbarHostState,
+    user: String
```

Comment:
can you rename this to `loggedInUser` or something like that. 100 user/username variables confuse the logged in user and the selected user for which we are explicitly viewing profile of.